ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -220,30 +222,31 @@ export const AccessRenewal = fp.flow(withProfileErrorModal)((spinnerProps: WithS
     onMount();
   }, []);
 
+  const maybeHeader = cond(
```

Comment:
extracted from old line 226.  no changes except for the name of `renewalCompleteAndNotExpiring`